node/src/actors/chain_manager/mod.rs

```diff
@@ -970,7 +970,25 @@ impl ChainManager {
         })
         .map_err(|e, _, _| log::error!("Superblock building failed: {:?}", e))
         .and_then(move |(block_headers, last_hash), act, _ctx| {
-            let ars_members = &act.chain_state.last_ars;
+            let chain_info = act.chain_state.chain_info.as_ref().unwrap();
+            let x;
+            let ars_members = {
+                // Before reaching the epoch activity_period + collateral_age the bootstrap committee signs the superblock
+                if act.current_epoch.unwrap()
+                    > chain_info.consensus_constants.collateral_age
+                        + chain_info.consensus_constants.activity_period
+                {
+                    &act.chain_state.last_ars
+                } else {
+                    x = chain_info
+                        .consensus_constants
+                        .bootstrapping_committee
+                        .iter()
+                        .map(|add| add.parse().unwrap())
+                        .collect();
+                    &x
+                }
+            };
```

Comment:
By making `ars_members` not a reference (but referencing later when using it as an argument), we can get rid of the `x` binding altogether.